src/layouts/login-form.html

```diff
@@ -50,7 +50,7 @@
       <img src="/static/icons/favicon-192x192.png" height="192" />
       <a href="#" id="hideKeyboardOnFocus"></a>
       <div class='interact'>
-        <div id='loginform' hidden$="[[showLoading]]">
+        <form id='loginform' hidden$="[[showLoading]]">
```

Comment:
Damnit, forgot I made the PR from my master branch, I tested that change yesterday at home (after making it at work) and it didn't work, so needs some more work.
Ill remove this change later today.